src/api/spec/jobs/update_released_binaries_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'rails_helper'
+
+RSpec.describe UpdateReleasedBinaries, vcr: true do
+  let(:project) { create(:project_with_repository) }
+  let(:repository) { project.repositories.first }
+  let(:event) { Event::Packtrack.new('project' => project.name, 'repo' => repository.name, 'payload' => 'fake_payload') }
```

Comment:
This could be `project: project.name, repo: repository.name, payload: 'fake_payload'`